### ISSUE TITLE
Error while deploying the Blog Generation application to Hugging Face

### ISSUE THREAD

**Issue by mrunankpawar** [krishnaik06/Complete-Langchain-Tutorials#8]
As we were running the model in our local, so we provided the path of llama model that we downloaded, but while deploying it on Hugging Face how can we actually get it done?

Can anyone please help?

Added screenshot of the error below. 👇

<img width="903" alt="Screenshot 2024-01-25 at 8 27 19 AM" src="https://github.com/krishnaik06/Complete-Langchain-Tutorials/assets/65391854/a28cd379-0e17-4772-9f97-ce8e2c730c66">

**Comment by dame-cell** [comment]
This happens because Llama is a gated repo as in you need to login to hugging face first which requires a huggingface account.

**Comment by mrunankpawar** [comment]
Yes, I understood that thing. I have gained access but I was wondering that how can we deploy it?

**Comment by 250519** [comment]
You can upload the llama model file in files of your respective space and also change the directory path in the app.py file too, it would run perfect!
